src/api/opportunity/content-types/opportunity/schema.json

```diff
@@ -55,6 +55,33 @@
       "relation": "manyToMany",
       "target": "api::project.project",
       "inversedBy": "opportunities"
+    },
+    "roleCategory": {
+      "type": "enumeration",
+      "enum": [
+        "Product / UX",
+        "Development",
+        "QA",
+        "Operations"
+      ],
+      "required": true
+    },
+    "roleType": {
+      "type": "enumeration",
+      "enum": [
+        "Product Lead",
+        "UX Designer",
+        "UX Researcher",
+        "Information Architect",
+        "Lead Developer",
+        "Back-End Developer",
+        "Front-End Developer",
+        "QA Lead",
+        "QA Tester",
+        "Volunteer Coordinator",
+        "Social Media Manager"
+      ],
+      "required": true
```

Comment:
What would happen to existing data that doesn't have this field? Should we add a default and then we can fix it manually. https://docs.strapi.io/user-docs/content-type-builder/configuring-fields-content-type#-enumeration says we can have a default under `Advanced settings`.

Reply:
Changes Done as suggested